src/ontology/cl-edit.owl

```diff
@@ -21180,6 +21180,10 @@ SubClassOf(obo:CL_0005011 obo:CL_0005010)
 # Class: obo:CL_0005012 (multi-ciliated epithelial cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:CVS"^^xsd:string) obo:IAO_0000115 obo:CL_0005012 "A ciliated epithelial cell with many cilium."^^xsd:string)
+AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0005012 "MCC"@en)
```

Comment:
these look good!

The only minor thing is we don't usually use the language tags (@en). I don't think there is any harm in doing that though - @dosumis, @matentzn or @addiehl could you confirm?

@lubianat is your Protege set up automatically adding these?

Reply:
@nicolevasilevsky I think I set it once and then it stayed at the default, and I thought it would no harm :)

Reply:
I think its fine, but only do that for synonyms for now.